### PR TITLE
i18n(ja): Translate `nav.ts`

### PR DESCRIPTION
**Pull request by kyosuke**
Translated the sidebar menu into Japanese.

### CODE REVIEW

File: src/i18n/ja/nav.ts [hunk 1]
```diff
@@ -1,6 +1,50 @@
...
+	'guides/typescript': 'TypeScript',
+	'core-concepts/framework-components': 'UIフレームワーク',
+
+	// Reference
+	reference: 'リファレンス',
+	'reference/configuration-reference': '設定方法',
+	'reference/cli-reference': 'CLI',
+	'reference/api-reference': 'ランタイムAPI',
+	'reference/integrations-reference': 'インテグレーションAPI',
+	'reference/adapter-reference': 'アダプターAPI',
```
Comment (yuto343) [inline src/i18n/ja/nav.ts:46]: ここ英語ではexperimentalとついているの、考慮しなくて大丈夫ですかね？

**Review by yuto343 — COMMENTED** [review]
@kyosuke レビューしました！

File: src/i18n/ja/nav.ts (hunk 1 above)
Comment (delucis) [reply src/i18n/ja/nav.ts:46]: @yuto343 Thanks so much for reviewing! @kyosuke also has a few other translation PRs open if you have time to have a look? We’d really appreciate it ❤️

Comment (yuto343) [reply src/i18n/ja/nav.ts:46]: @delucis 
thanks replying!
OK, now it looks good to me.

**Review by yuto343 — APPROVED** [review]
LGTM!

**Review by delucis — APPROVED** [review]
@kyosuke @yuto343 Thanks team! 💜

**Pull request merged**